llvm/test/DebugInfo/CAS/AArch64/test_relocation_zero.ll

```diff
@@ -0,0 +1,64 @@
+;RUN: llc -O0 --filetype=obj --cas-backend  --cas=%t/cas  --mccas-casid %s -o %t/test.id
+;RUN: llvm-cas-dump --cas=%t/cas --casid-file %t/test.id --hex-dump | FileCheck %s
+
+;CHECK: mc:debug_info_cu llvmcas://{{[0-9a-z]+}}
+;CHECK-NEXT: 0x3c 0x00 0x00 0x00 0x04 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x08 0x01 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x0c 0x00 0x6a 0x00 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x93 0x00 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x00 0x00 0x00 0x00 0x04 0x00
+;CHECK-NEXT: 0x00 0x00 0x02 0x00 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x00 0x04 0x00 0x00 0x00 0x01
+;CHECK-NEXT: 0x6f 0xb8 0x00 0x00 0x00 0x01 0x01 0x00
+;CHECK-NEXT: mc:debug_info_cu llvmcas://{{[0-9a-z]+}} 
+;CHECK-NEXT: 0x56 0x00 0x00 0x00 0x04 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x08 0x01 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x0c 0x00 0x6a 0x00 0x00 0x00 0x3c 0x00
+;CHECK-NEXT: 0x00 0x00 0x93 0x00 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x00 0x00 0x00 0x00 0x18 0x00
+;CHECK-NEXT: 0x00 0x00 0x02 0x00 0x00 0x00 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x00 0x18 0x00 0x00 0x00 0x01
+;CHECK-NEXT: 0x6f 0xbc 0x00 0x00 0x00 0x01 0x02 0x52
+;CHECK-NEXT: 0x00 0x00 0x00 0x03 0x02 0x91 0x0c 0xc4
+;CHECK-NEXT: 0x00 0x00 0x00 0x01 0x02 0x52 0x00 0x00
+;CHECK-NEXT: 0x00 0x00 0x04 0xc0 0x00 0x00 0x00 0x05
+;CHECK-NEXT: 0x04 0x00
```

Comment:
Some comments here would be nice, otherwise this test can soon turn into one of those that, whenever it fails,  people will just copy paste whatever new bytes show up :)

For example, which parts of the CU are relocations expected to be zero?